cpp/src/io/parquet/stats_filter_helpers.hpp

```diff
@@ -311,6 +311,8 @@ class stats_expression_converter : public stats_columns_collector {
 
  private:
   ast::tree _stats_expr;
+  cudf::numeric_scalar<bool> _always_true_scalar{true};
```

Comment:
This constructor is missing a stream parameter.
Is there a stream test for this code?